src/instructlab/cli/data/generate.py

```diff
@@ -165,6 +165,20 @@
 @click.option(
     "-dt", "--detached", is_flag=True, help="Run ilab data generate in the background"
 )
+@click.option(
+    "--student-model-id",
+    # help="ID of the custom model you want to use as the student model.",
+    default=None,
```

Comment:
do we want this to default to the model in the train part of the cfg or are we getting rid of those defaults?

Reply:
We want to leave this blank for now.